test/data.js

```diff
@@ -29,5 +29,49 @@ describe('Tech radar', function(){
 
         assert.equal(viewModel[0].values[0].status, 'Assessing');
         });
+
+
+         it('should order the tech by status', function(){
+
+        var statusList = [
+            {
+            id : 1,
+            name : 'Assessing'
+            },{
+            id : 2,
+            name : 'Trialling'
+            },
+            {
+            id : 3,
+            name : 'Adopted'
+            }
+        ];
+
+        var categories = [
+            {               
+            values : [
+                { 
+                    name: "Second",
+                    status : 2
+                },
+                 { 
+                    name: "Third",
+                    status : 3
+                },
+                 { 
+                    name: "First",
+                    status : 1
+                }
+            ]
+            }
+        ];
+        
+
+        var viewModel = dataMod(categories, statusList);
+
+        assert.equal(viewModel[0].values[0].status, 'Assessing');
```

Comment:
I think this would make more sense using 'First', 'Second', 'Third' as opposed to the status name.  That way we are clearly asserting on the order of the items, as opposed to there status, if you get what i mean